src/onegov/fsi/models/course_event.py

```diff
@@ -20,15 +21,29 @@
 from onegov.fsi.models.course_attendee import CourseAttendee
 from onegov.fsi.models.course_subscription import CourseSubscription
 from onegov.fsi.models.course_subscription import subscription_table
-from onegov.search import ORMSearchable
-
+from onegov.search import ORMSearchable, Searchable
 
 from typing import TYPE_CHECKING
 if TYPE_CHECKING:
     from .course import Course
     from .course_notification_template import (
-        CancellationTemplate, CourseNotificationTemplate, InfoTemplate,
-        ReminderTemplate, SubscriptionTemplate
+        CancellationTemplate,
+        CourseNotificationTemplate,
+        InfoTemplate,
+        ReminderTemplate,
+        SubscriptionTemplate
+    )
+
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from .course import Course  # noqa: F811
+    from .course_notification_template import (  # noqa: F811
+        CancellationTemplate,   # noqa: F811
+        CourseNotificationTemplate,  # noqa: F811
+        InfoTemplate,  # noqa: F811
+        ReminderTemplate,   # noqa: F811
+        SubscriptionTemplate  # noqa: F811
```

Comment:
This looks like a merge conflict resolution error, you don't need this if statement twice...